### DISCUSSION TITLE
Problem with M3U output

### DISCUSSION THREAD

**Discussion by ddeslandes1** [euzu/tuliprox#56] [Q&A] [answered]
I am able to filter out the list of channels,VOD,series but I would like to do is save a "valid" m3u file, at the moment it saves with null values everywhere, not sure if that is because my windows is set in french language 
  targets:
    - name: all_channels
      output:
        - type: m3u
          filename: playlist_1.m3u
      processing_order: frm
      options:
          ignore_logo: true
      sort:
          order: asc
      filter: "!ALL_CHAN!"    
any idea ?

**Comment by euzu** [comment, marked as answer]
@ddeslandes1  this is the 2.0.0 version, which does not support  plain text files any more.
Ok i totally forgot this case.

I will try to add cli arguments  to output m3u playlist as plain file in non server mode.

What you can do now is,  run m3u-filter in server mode and access the playlist through the api.  But to do this you need to  configure api-proxy.yml  to create user and give access to the target.

Sorry i will try to fix this issue.

**Reply by ddeslandes1** [reply]
can you delete image from my 1st original post please

**Reply by euzu** [reply]
i cant find the image, if it included  sensitive information then you should change your credentials or request a change  from.provider.

**Reply by ddeslandes1** [reply]
Can you just delete this discussion then please

**Reply by euzu** [reply]
You have removed the image from this post. It is not accessible without the direct link. It is not possible to delete posts.
Github provides a way to delete images with their url, but it does not work.

**Comment by euzu** [comment]
@ddeslandes1  the current main-branch has the changes. 
The `filename`attribute is now optional for m3u target outputs. If you give a filename, the plain m3u playlist will be saved under the filename.
